FinSearchUnified/Tests/Helper/StaticHelperTest.php

```diff
@@ -593,4 +595,181 @@ public function testParsesSmartDidYouMeanData(
 
         StaticHelper::setSmartDidYouMean($xmlResponse);
     }
+
+    /**
+     * @dataProvider discountFilterProvider
+     * @dataProvider priceFilterProvider
+     *
+     * @param array $filterData
+     * @param array $parameters
+     * @param bool $expectedFacetResult
+     * @param string $expectedMinField
+     * @param string $expectedMaxField
+     *
+     * @throws Zend_Http_Client_Exception
+     */
+    public function testCreateRangeSliderFacetMethod(
+        array $filterData,
+        array $parameters,
+        $expectedFacetResult,
```

Comment:
Rename this to `$expectedFacetState` since it is a boolean and not an actual facet.